packages/annotation/src/core/toolOperation/cuboidToggleButtonClass.ts

```diff
@@ -0,0 +1,233 @@
+import { ECuboidDirection } from '@/constant/annotation';
+import cuboidFAB from '@/assets/attributeIcon/icon_cuboidFAB.svg';
+import cuboidMore from '@/assets/attributeIcon/icon_cuboidMore.svg';
+import cuboidRight from '@/assets/attributeIcon/icon_cuboidRight.svg';
+import cuboidLeft from '@/assets/attributeIcon/icon_cuboidLeft.svg';
+import cuboidTop from '@/assets/attributeIcon/icon_cuboidTop.svg';
+
+interface IBaseInfo {
+  cuboidButtonMove: (type: 'in' | 'out') => void;
+  toggleGlossySide: (direction: ECuboidDirection) => void;
+}
+
+interface ITextAttributeProps {
+  container: HTMLElement;
+}
+
+const MORE_ICON_LISTS = [
+  {
+    icon: cuboidRight,
+    id: 'cuboidRight',
+  },
+  {
+    icon: cuboidLeft,
+    id: 'cuboidLeft',
+  },
+  {
+    icon: cuboidTop,
+    id: 'cuboidTop',
+  },
+];
+
+export default class CuboidToggleButtonClass {
+  private container: HTMLElement;
+
+  private direction: ECuboidDirection;
+
+  private _cuboidButtonDOM?: HTMLElement;
+
+  private _cuboidFABDOM?: HTMLElement; // Switch icons forward and back
+
+  private isFrontSide: boolean; // Is it front
+
+  private _cuboidMoreDOM?: HTMLElement; // More icons
+
+  private _cuboidMoreListDOM?: HTMLElement; // More icon lists
+
+  private cuboidButtonMove: (type: 'in' | 'out') => void;
+
+  private onToggleGlossySide: (direction: ECuboidDirection) => void;
+
+  constructor(props: ITextAttributeProps & IBaseInfo) {
+    const { container, cuboidButtonMove, toggleGlossySide } = props;
+    this.container = container;
+    this.direction = ECuboidDirection.Front;
+    this.isFrontSide = true;
+    this.cuboidButtonMove = cuboidButtonMove;
+    this.onToggleGlossySide = toggleGlossySide;
+    this._cuboidButtonDOM = this.initCuboidButtonDOM();
+    this._cuboidFABDOM = this.initcuboidFABDOM(cuboidFAB);
+    this._cuboidMoreDOM = this.initcuboidMoreDOM(cuboidMore);
+    this._cuboidMoreListDOM = this.initcuboidMoreListDOM();
+
+    // 绑定到 dom 上
+    this.appendToContainer();
+  }
+
+  public appendToContainer() {
+    if (!this._cuboidButtonDOM || !this._cuboidFABDOM || !this._cuboidMoreDOM) {
+      return;
+    }
+
+    this.container.appendChild(this._cuboidButtonDOM);
+    this._cuboidButtonDOM.appendChild(this._cuboidFABDOM);
+    this._cuboidButtonDOM.appendChild(this._cuboidMoreDOM);
+  }
+
+  public initCuboidButtonDOM() {
+    const _cuboidButtonDOM = document.createElement('div'); // Top floor
+    _cuboidButtonDOM.setAttribute('id', 'box');
```

Comment:
Don't use string directly. Needs to use like `LABELBEE_CUBOID_BOX` which is nearly unique